docs/testbenches/ip_based/util_pack/index.rst

```diff
@@ -0,0 +1,277 @@
+.. _util_pack:
+
+Util Pack
+================================================================================
+
+Overview
+-------------------------------------------------------------------------------
+
+The purpose of this testbench is to validate the serial interface functionality
```

Comment:
The purpose is to validate the serial interface functionality? or it is a leftover from a previous example?